pkgs/applications/graphics/dia/default.nix

```diff
@@ -13,6 +13,10 @@ stdenv.mkDerivation {
     sha256 = "1fyxfrzdcs6blxhkw3bcgkksaf3byrsj4cbyrqgb4869k3ynap96";
   };
 
+  patches = [
+    ./CVE-2019-19451.patch
```

Comment:
Can we fetch the patch from gnome instead of vendoring it?

Reply:
That patch does not apply